changelog.md

```diff
@@ -5,6 +5,7 @@
 ### Changes
 
 - [#3959](https://github.com/ignite/cli/pull/3959) Remove app name prefix from the `.gitignore` file
+- [#3962](https://github.com/ignite/cli/pull/3962) Rename all RPC endpoints and autocli commands generated for `map`/`list`/`single` types 
```

Comment:
Maybe:

```suggestion
- [#3962](https://github.com/ignite/cli/pull/3962) Rename list and get RPC endpoints and commands for generated `map`, `list`, `single` types 
```

Reply:
For RPC they are not `list-foo` and `show-foo`. So for example if we have a type named Foo:

- query Foo -> GetFoo
- query FooAll -> ListFoo
- cli `show-foo` -> `get-foo`
- cli `list-foo` without any changes

I couldn't find a way to fit all this explanation in a user-friendly way in one line.